src/index.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @description Check if provided string is an `IdentifierName` as specified in [ECMA262 edition 5.1 section 7.6](https://262.ecma-international.org/5.1/#sec-7.6)
+ *
+ * @example
+ * ```js
+ * import isEs5IdentifierName from "@prettier/is-es5-identifier-name";
+ *
+ * isEs5IdentifierName("fisker");
+ * // -> true
+ *
+ * isEs5IdentifierName("16");
+ * // -> false
```

Comment:
Let's use `isEs5IdentifierName("prettier");` as example (sync readme.md please).  I think one case is good enough.